.gitignore

```diff
@@ -2,3 +2,4 @@
 */target/
 **/*.rs.bk
 Cargo.lock
+.idea
```

Comment:
We generally encourage ppl to ignore folders like this in the global git configuration.

Reply:
> We generally encourage ppl to ignore folders like this in the global git configuration.

indeed